web/src/beta/features/PluginPlayground/Plugins/usePlugins.ts

```diff
@@ -170,42 +170,77 @@ export default () => {
     [selectedPlugin, selectedFileId]
   );
 
-  const handleFileUpload = useFileInput((fileList) => {
-    const file = fileList?.[0];
-    if (!file) {
-      return;
-    }
-    const result = validateFileTitle(
-      file.name,
-      selectedPlugin.files.map((f) => f.title)
-    );
-    if (!result.success) {
-      setNotification({ type: "error", text: result.message });
-      return;
-    }
+  const handleFileUpload = useFileInput(
```

Comment:
```suggestion
  const handlePluginImport = useFileInput(
```
Suggest to use a more clear naming here.

Reply:
Thanks for spotting that. Done